src/app/util/ember-compatibility-functions.cpp

```diff
@@ -248,6 +248,34 @@ bool ServerClusterCommandExists(const ConcreteCommandPath & aCommandPath)
 
 namespace {
 
+CHIP_ERROR ReadClusterDataVersion(const EndpointId & aEndpointId, const ClusterId & aClusterId, DataVersion & aDataVersion)
+{
+    EmberAfCluster * cluster = emberAfFindCluster(aEndpointId, aClusterId, CLUSTER_MASK_SERVER);
+    if (cluster == nullptr)
+    {
+        return CHIP_ERROR_NOT_FOUND;
+    }
+    aDataVersion = *(cluster->version);
+    return CHIP_NO_ERROR;
+}
+
+void IncreaseClusterDataVersion(const EndpointId & aEndpointId, const ClusterId & aClusterId)
+{
+    EmberAfCluster * cluster = emberAfFindCluster(aEndpointId, aClusterId, CLUSTER_MASK_SERVER);
+    if (cluster == nullptr)
+    {
+        ChipLogError(DataManagement, "Path: Endpoint %" PRIx16 ", Cluster %" PRIx32 " not found in IncreaseClusterDataVersion!",
+                     aEndpointId, aClusterId);
+    }
+    else
+    {
+        (*(cluster->version))++;
```

Comment:
Again, this should be done via a method on EmberAfCluster.  We're not implementing in C; let's not be limited to the things C could do here.

Reply:
EmberAfCluster is typedef struct...we cannot add member function inside EmberAfCluster